static/css/dashboard.css

```diff
@@ -211,28 +211,16 @@ body.deployment-devsite .navbar-brand img {
 
 
 /* Major styling for different deployments */
-body.deployment-laptop {
-  background-image: linear-gradient(45deg, #f7f7d7 2.83%, rgba(255, 255, 255, 0) 2.83%, rgba(255, 255, 255, 0) 50%, #f7f7d7 50%, #f7f7d7 52.83%, rgba(255, 255, 255, 0) 52.83%, rgba(255, 255, 255, 0) 100%);   
-  background-size: 100vw 100vh;
-  background-repeat: no-repeat;
-}
-
 body.deployment-laptop #navbar-brand
 {
-  background-color: #f7f7d7 !important;
+  background-color: #fff3cd !important;
```

Comment:
Oh boy.  `!important` declarations in CSS are a major headache and slow page render performance.  We will have to tackle these at some point.

Reply:
Sure.  They were already present, so I just left them as they are.  I'll link this comment to the CSS related issue as well.